backend/pricing/pricing.go

```diff
@@ -116,10 +117,12 @@ func TypeToMemberLimit(planType backend.PlanType, unlimitedMembers bool) *int {
 	}
 }
 
-func TypeToQuota(planType backend.PlanType) int {
+func TypeToSessionsLimit(planType backend.PlanType) int {
 	switch planType {
 	case backend.PlanTypeFree:
 		return 500
+	case backend.PlanTypeLite:
```

Comment:
are we renaming lite -> basic and basic -> essentials?

Reply:
yep, I've made the changes in the frontend but for our code I'm not sure if it's better to change these enum values and migrate things in the DB/stripe lookup keys or just leave as is. Just seems a little error prone

Reply:
sounds good!